src/app/(serverOnly)/sr/page.tsx

```
@@ -0,0 +1,11 @@
+import React from "react";
+
+import serverSideFunction from "@/utils/server-utils";
+
+type Props = {};
+
+export default function ServerPage({}: Props) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)